utils/databake/src/primitives.rs

```diff
@@ -193,6 +206,16 @@ fn result() {
 }
 
 macro_rules! tuple {
+    ($ty:ident, $ident:ident) => {
+        impl<$ty> Bake for ($ty,) where $ty: Bake {
```

Comment:
nit: can we just redirect to the same macro with a comma tacked on

Reply:
Not sure how to do that without it being too convoluted. What did you have in mind?

Reply:
wait never mind I misunderstood this macro, I thought it was recursive